homework-submission/my-app/src/Components/button.js

```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react';
+
+class Button extends Component {
+  render() {
+    return (
+      <button
+        onClick={this.props.onClick}
+        type={this.props.type}
+        className={this.props.className}
+        key={this.props.key}
```

Comment:
you may have another look here, the reason: 
 - you have specified a key property to pass it to this component, inside the component this button doesn't receive any property called `key` because it's a stricted prop-name for the react framework for using it in an arrays.